lib/index.ts

```diff
@@ -89,31 +97,31 @@ export class DashLine {
             })
             this.activeTexture = texture
         } else {
-            this.graphics.lineStyle({
+            this.graphics.setStrokeStyle({
```

Comment:
This method should be changed to `stroke()`! 

`setStrokeStyle` operates on graphicsContext in v8, NOT on graphics. The v8 docs incorrectly state that this method is the replacement for `lineStyle`.